index.html

```diff
@@ -55,17 +56,17 @@ <h3>Welcome!</h3>
   <div class="flex-wrap">
 
           <section id="gal" class="gal">
-          	 <a href="Gallaries/art.html" target="blank"> <h2>Gallery</h2>
+          	 <a href="Galleries/art.html" target="blank"> <h2>Gallery</h2>
               </a>
-              <p>
+              <p><b>Fine &amp; Crude Artwork</b>
```

Comment:
added bold in paragraph, which good sub-header, as repeats on webpage and a range of content description.